pytensor/graph/op.py

```diff
@@ -289,7 +289,11 @@ def __call__(self, *inputs: Any, **kwargs) -> Variable | list[Variable]:
 
         """
         return_list = kwargs.pop("return_list", False)
+        name = kwargs.pop("name", None)
         node = self.make_node(*inputs, **kwargs)
+        if isinstance(node.outputs, list):
```

Comment:
This will always be a list. However I think we should only assign name when there's a single output or a default output

Reply:
If my understanding is correct

If `node.outputs` has a length of more than 1, then they should be called `f"{name}_{i}"`
Here `i` would be the `default_output`(which is specified in Multi-output Ops)

Reply:
Not quite what I was thinking. Multi-output nodes can have a default output, which is the only thing users usually see. We should pass name directly to either to the default output of a multi-output node or the only output of a single-output node.

But for multi-output nodes without a default output we shouldn't. For those I suggest adding a numerical suffix, 0 for the first, 1 for the second, and so on